<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slee Practice</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_search">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searcher"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Search">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_sortAZ">
        <property name="text">
         <string>A-Z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_sortZA">
        <property name="text">
         <string>Z-A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_sortDate">
        <property name="text">
         <string>Creation Date</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="MyListView" name="listView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyListView</class>
   <extends>QListView</extends>
   <header>MyListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
